flexibleengine/provider.go

```diff
@@ -320,6 +320,8 @@ func Provider() *schema.Provider {
 			"flexibleengine_modelarts_datasets":         modelarts.DataSourceDatasets(),
 			"flexibleengine_modelarts_dataset_versions": modelarts.DataSourceDatasetVerions(),
 
+			"flexibleengine_obs_bucket_object": obs.DataSourceObsBucketObject(),
+
```

Comment:
why provider.go have only one data source name? this field should have two resource and two data source

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/ef837129-2a59-4753-b8cd-32feed1ba8fb)
good catch, I lose these resources or data sources accidently,  and I check the ut results again, the results are right.